src/components/Header.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import {Nav, NavItem, NavLink} from 'reactstrap';
+
+class Header extends Component{
+  render(){
+    return(
+      <>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
Pretty sure this needs to be capital "N" Nav

Reply:
Capitalizing the nav tags to Nav caused the bootstrap code to not render properly and made styling and links disappear. Kept the tags lower cased.
<img width="628" alt="Screen Shot 2022-04-05 at 3 37 25 PM" src="https://user-images.githubusercontent.com/88990920/161863387-9bdd2ca4-c70e-4187-b248-cfcd7ebd7fc4.png">